test/integrations/destinations/smartly/processor/track.ts

```diff
@@ -0,0 +1,75 @@
+import { destination } from '../commonConfig';
+
+export const trackTestData = [
+  {
+    name: 'smartly',
+    description: 'Test 0',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            destination,
+            message: {
+              event: 'Add to cart',
+              properties: {
+                platform: 'meta',
+                ad_unit_id: '228287',
+                ad_interaction_time: '1650626278',
+                email: 'eventIdn01@sample.com',
+              },
+              type: 'track',
+              userId: 'eventIdn01',
+            },
+            metadata: {
+              destinationId: 'dummyDestId',
+              jobId: '1',
+            },
+          },
+        ],
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            metadata: {
+              destinationId: 'dummyDestId',
+              jobId: '1',
+            },
+            output: {
+              version: '1',
+              type: 'REST',
+              method: 'POST',
+              endpoint: 'https://s2s.smartly.io/events',
+              headers: {},
+              params: {},
+              body: {
+                JSON: {
+                  events: [
+                    {
+                      platform: 'meta',
+                      ad_unit_id: '228287',
+                      ad_interaction_time: '1650626278',
+                      conversions: '1',
```

Comment:
please add doc reference in support of default value 1 for conversions

Reply:
Smartly Doc for reference : https://support.smartly.io/hc/en-us/articles/4406049685788-S2S-integration-API-description#01H8HBXZF6WSKSYBW1C6NY8A88